addon-test-support/setup-console-logger.ts

```diff
@@ -0,0 +1,8 @@
+import QUnit from 'qunit';
+import { printResults } from './logger';
+
+export function setupConsoleLogger() {
```

Comment:
I think we'll need to add some docs to the README for this (and possibly some API docs).

Reply:
I mentioned this in [my comment above](https://github.com/ember-a11y/ember-a11y-testing/pull/214#discussion_r485843742). I'm going to do a full docs pass on the entire repo once all these changes are merged in.

Reply:
Sounds good, mind double checking the meta issue to make sure we have this logged?

Reply:
For sure, and it is!

Reply:
I updated the line item to be a bit more precise to what we're needing, namely a README update for the API and functional changes.